Dockerfile

```
@@ -25,15 +25,21 @@ ENV PORT=5000
 ENV PUID=1000
 ENV PGID=1000
 
-# Install su-exec (for privilege dropping), shadow (for usermod/groupmod), and
-# Python + Apprise for local CLI notifications.
-RUN apk add --no-cache su-exec shadow python3 py3-pip && \
+# Install su-exec (for privilege dropping), shadow (for usermod/groupmod),
+# unrar (RAR archive extraction, community repo), and Python + Apprise for
+# local CLI notifications.
+RUN apk add --no-cache su-exec shadow unrar python3 py3-pip && \
```

Hadolint (2.14.0)

[warning] Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>` or `pip install --requirement <requirements file>`

(DL3013)